python/feast_spark/job_service.py

```diff
@@ -326,6 +362,16 @@ def ensure_stream_ingestion_jobs(client: Client, all_projects: bool):
         if all_projects
         else [client.feature_store.project]
     )
+    filtered_projects = []
+    if client.config.exists(opt.WHITELISTED_PROJECTS):
+        whitelisted_projects = client.config.get(opt.WHITELISTED_PROJECTS)
+        if whitelisted_projects:
+            whitelisted_projects = whitelisted_projects.split(",")
+            filtered_projects = [
```

Comment:
why do we need this extra variable `filtered_projects`. Can't we just overwrite `projects` here?